src/harness/testcases/WINNF_FT_S_QPR_testcase.py

```diff
@@ -491,3 +494,342 @@ def test_WINNF_FT_S_QPR_8(self, config_filename):
 
     # CHECK: Total EIRP of all approved grants is <= 49.15 dBm
     self.assertLessEqual(total_eirp_dbm, 49.15)
+
```

Comment:
tiniest of issues...this code is added at the end of the file so other QPR TCs with a higher number 6,7,8 will be above it.  Please move so it'll be above QPR6.  I run it and the config generated fine and CBSDs registered/granted etc....looking through the specifics of the config now

Reply:
Done :)